donate/settings/environment.py

```diff
@@ -0,0 +1,95 @@
+import os
+import environ
+
+CSP_DEFAULT_SRC = ("'self'")
+CSP_SCRIPT_SRC = ("'self'", '\'unsafe-inline\'', '\'unsafe-eval\'',
```

Comment:
if `"'...'"` works, we should probably use that for the rest of the settings too, rather than `'\'...\''`.